R/structural.properties.R

```diff
@@ -2508,6 +2507,14 @@ unfold_tree <- function(graph, mode = c("all", "out", "in", "total"), roots) {
 #' @author Gabor Csardi \email{csardi.gabor@@gmail.com}
 #' @export
 #' @keywords graphs
+#' @section Normalization methods:
+#'
+#' - "unnormalized": Unnormalized Laplacian, \eqn{L = D - A}.
+#' - "symmetric": Symmetric normalized Laplacian,
+#' \eqn{L = I - D^(-1/2) A D^(-1/2)}.
```

Comment:
In LaTeX, we cannot use `x^abc`, as that renders as $x^abc$. We must write $x^{abc}$. If we do so, the code may become too weird to be human-readable, so we must also provide a human-readable "plain text" version to roxygen2.

```suggestion
#' \eqn{L = I - D^{-\frac{1}{2}} A D^{-\frac{1}{2}}}{L = I - D^(-1/2) A D^(-1/2)}.
```